retro/utils/stats.py

```diff
@@ -351,4 +336,61 @@ def estimate_from_llhp(llhp, meta=None, per_dim=False, prob_weights=True):
             estimate['weighted_mean'][col] = weighted_mean
             estimate['weighted_median'][col] = weighted_median
 
+
+    for angle in ['', 'track_', 'cascade_']:
+        if angle+'zenith' in columns and angle+'azimuth' in columns:
+            # currently double work, but for testing purposes
+            # idea, calculated the medians on the sphere for az and zen combined
+            if per_dim:
+                # no can do in this case
+                return estimate
+
+            if prob_weights is None and prior_weights is None:
+                weights = None
+            else:
+                if prob_weights is None:
+                    weights = np.ones(len(llhp))
+                else:
+                    weights = np.copy(prob_weights)
+            if prior_weights is not None:
+                weights *= prior_weights
+
+            az = llhp[angle+'azimuth']
+            zen = llhp[angle+'zenith']
+            llh_cut = llhp['llh'] > np.max(llhp['llh']) - 15.5
```

Comment:
add comment anywhere magic numerical constants appear and/or define a `CONST` at the top of the file for clarity what magic numbers are used in a module